packages/components/date-picker/src/date-picker-com/panel-date-pick.vue

```diff
@@ -361,12 +361,34 @@ type Shortcut = {
   onClick?: (ctx: Omit<SetupContext, 'expose'>) => void
 }
 
+const arrangeDatesShortcuts = (selectedValue: Dayjs) => {
+  if (!isArray(props.parsedValue)) {
+    return emit([selectedValue], true)
+  }
+
+  const index = props.parsedValue.findIndex(
+    (date) =>
+      dayjs(date).format('YYYY-MM-DD') ===
+      dayjs(selectedValue).format('YYYY-MM-DD')
+  )
+  const dates =
+    index > -1
+      ? castArray(props.parsedValue).filter((d, i) => i !== index)
+      : (castArray(props.parsedValue) as Dayjs[]).concat([selectedValue])
```

Comment:
Line 365 has already judged whether it is an array, and castArray should not be needed here.

Reply:
Oh, that's right